### ISSUE TITLE
The game crash with compressor

### ISSUE THREAD

**Issue by aroxy** [TeamPneumatic/pnc-repressurized#102]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.12.2
### Forge Version
14.23.1.2590
### Mod Version
1.12.2-0.3.0-124
### Describe your problem, including steps to reproduce it
The game crash when y pose a compressor

**Comment by TeamSpen210** [comment]
More precisely, placing any block gives an `AbstractMethodError` from the superclass call in `TileEntityTickableBase.update()`. Strangely it does not happen in deobfuscated mode.

```
---- Minecraft Crash Report ----
// Would you like a cupcake?

Time: 1/18/18 9:46 AM
Description: Ticking block entity

java.lang.AbstractMethodError: me.desht.pneumaticcraft.common.tileentity.TileEntityTickableBase.func_73660_a()V
	at me.desht.pneumaticcraft.common.tileentity.TileEntityPneumaticBase.func_73660_a(TileEntityPneumaticBase.java:36)
	at me.desht.pneumaticcraft.common.tileentity.TileEntityProgrammableController.func_73660_a(TileEntityProgrammableController.java:99)
	at net.minecraft.world.World.func_72939_s(World.java:1832)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:612)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:765)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at me.desht.pneumaticcraft.common.tileentity.TileEntityPneumaticBase.func_73660_a(TileEntityPneumaticBase.java:36)
	at me.desht.pneumaticcraft.common.tileentity.TileEntityProgrammableController.func_73660_a(TileEntityProgrammableController.java:99)

-- Block entity being ticked --
Details:
	Name: pneumaticcraft:programmablecontroller // me.desht.pneumaticcraft.common.tileentity.TileEntityProgrammableController
	Block type: ID #292 (tile.programmable_controller // me.desht.pneumaticcraft.common.block.BlockProgrammableController)
	Block data value: 0 / 0x0 / 0b0000
	Block location: World: (1291,56,156), Chunk: (at 11,3,12 in 80,9; contains blocks 1280,0,144 to 1295,255,159), Region: (2,0; contains chunks 64,0 to 95,31, blocks 1024,0,0 to 1535,255,511)
	Actual block type: ID #292 (tile.programmable_controller // me.desht.pneumaticcraft.common.block.BlockProgrammableController)
	Actual block data value: 0 / 0x0 / 0b0000
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1832)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:612)

-- Affected level --
Details:
	Level name: Copy of New World
	All players: 1 total; [EntityPlayerMP['TeamSpen210'/319, l='Copy of New World', x=1291.50, y=56.00, z=155.50]]
	Chunk stats: ServerChunkCache: 625 Drop: 0
	Level seed: -5828980689257625989
	Level generator: ID 01 - flat, ver 0. Features enabled: true
	Level generator options: 3;minecraft:bedrock,3*minecraft:stone,52*minecraft:sandstone;2;
	Level spawn location: World: (1287,4,158), Chunk: (at 7,0,14 in 80,9; contains blocks 1280,0,144 to 1295,255,159), Region: (2,0; contains chunks 64,0 to 95,31, blocks 1024,0,0 to 1535,255,511)
	Level time: 417 game time, 417 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 103277 (now: false), thunder time: 175811 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:765)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_102, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 752077408 bytes (717 MB) / 1124597760 bytes (1072 MB) up to 1908932608 bytes (1820 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms256m -Xmx2048m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.1.2590 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State     | ID             | Version          | Source                                            | Signature                                |
	|:--------- |:-------------- |:---------------- |:------------------------------------------------- |:---------------------------------------- |
	| UCHIJAAAA | minecraft      | 1.12.2           | minecraft.jar                                     | None                                     |
	| UCHIJAAAA | mcp            | 9.42             | minecraft.jar                                     | None                                     |
	| UCHIJAAAA | FML            | 8.0.99.99        | forge-1.12.2-14.23.1.2590-universal.jar           | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | forge          | 14.23.1.2590     | forge-1.12.2-14.23.1.2590-universal.jar           | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | pneumaticcraft | 1.12.2-0.3.0-124 | pneumaticcraft-repressurized-1.12.2-0.3.0-124.jar | None                                     |

	Loaded coremods (and transformers): 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Player Count: 1 / 8; [EntityPlayerMP['TeamSpen210'/319, l='Copy of New World', x=1291.50, y=56.00, z=155.50]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'
```

**Comment by desht** [comment]
Fixed, and v0.3.1 is being uploaded to Curseforge now.
